### DISCUSSION TITLE
Reset circuit

### DISCUSSION THREAD

**Discussion by kpishere** [slu4coder/Minimal-64x4-Home-Computer#30] [General]
I had one of these kicking around (it was from a washing machine :P ).  It is a reset circuit with a watchdog, not using that, just the reset.  Is a really nice addition to have that as now I can connect the USB and source it for power and it just starts up nicely.  The manual reset button still works fine.

![IMG_0636](https://github.com/user-attachments/assets/928862a7-0ec7-4942-81bb-ced2baa8dce5)

**Comment by nateo87** [comment]
Very cool. Makes me wonder if I could/should make a little 555 timer reset circuit for mine?

**Reply by kpishere** [reply]
Yeah, especially if you're connecting it to a CRT as there is never an invalid VGA signal in this case.  Well, I have not confirmed that, if there is, it only exists for under 200ms as power comes up.

**Reply by nateo87** [reply]
I am planning on hooking it up to a CRT at some point soon - I have a few AD724 chips which can take the VGA signal and convert it to NTSC composite. I can't help but wonder if I can somehow fudge the graphics a bit to trick NTSC into displaying color 🤔

**Reply by slu4coder** [reply]
Very nice idea. A 555 timer would be a valuable addition. The reset on the Minimal is -- well minimal :-)
